WiserTaskScheduler/AutoUpdater/Services/UpdateService.cs

```diff
@@ -353,12 +369,34 @@ private void RevertUpdate(WtsModel wts, ServiceController serviceController, Ver
             // Try to start the previous installed version again.
             serviceController.Start();
             serviceController.WaitForStatus(ServiceControllerStatus.Running);
-            EmailAdministrator(wts.ContactEmail, "WTS Auto Updater - Updating failed!", $"Failed to update WTS '{wts.ServiceName}' to version {versionToUpdateTo}, successfully restored to version {currentVersion}.<br/><br/>Error when updating:<br/>{updateException.ToString().ReplaceLineEndings("<br/>")}", wts.ServiceName);
+
+            var subject = "WTS Auto Updater - Updating failed!";
+            var message= $"Failed to update WTS '{wts.ServiceName}' to version {versionToUpdateTo}, successfully restored to version {currentVersion}.<br/><br/>Error when updating:<br/>{updateException}";
+            
+            logger.LogError(message);
+            InformPeople(wts,subject,message);
```

Comment:
I'm not going to reject the PR for this, but please separate parameters with spaces.